spec/models/miq_server_spec.rb

```diff
@@ -2,9 +2,13 @@
   include_examples ".seed called multiple times"
 
   it ".invoke_at_startups" do
+    MiqRegion.seed
     described_class::RUN_AT_STARTUP.each do |klass|
+      next unless klass.respond_to?(:atStartup)
       expect(klass.constantize).to receive(:atStartup)
     end
+
+    expect(Vmdb.logger).to receive(:log_backtrace).never
```

Comment:
That's funny... The reason I use this form is I'm used to typing `.to eq` that I type `.to not_receive` or similar  instead of`.not_to receive`.